.github/dependabot.yml

```diff
@@ -1,16 +1,21 @@
 version: 2
 updates:
-  - package-ecosystem: "bundler"
+  - package-ecosystem: bundler
     directory: "/"
     schedule:
-      interval: "daily"
-    open-pull-requests-limit: 0 # security only
-  - package-ecosystem: "docker"
+      interval: daily
+    open-pull-requests-limit: 10 # security only
```

Comment:
if we're changing this limit we should remove the comment, as it's no longer accurate: `0` means only security-related updates, anything `> 0` means all updates.